packages/regl-worldview/src/types/index.js

```diff
@@ -188,6 +188,7 @@ export type TriangleList = BaseShape & {
 
 export type PolygonType = BaseShape & {
   points: Point[],
+  color: Color,
```

Comment:
Just a note that we'll need to add `colors?` as well in the future, to enable individual vertex coloring